packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -355,6 +356,26 @@ workflowsController.patch(
 			);
 		}
 
+		const numberOfUsers = await Db.collections.User.count();
```

Comment:
I don't think we need to check the number of users since this should never be an issue; in fact we might be simply hiding the issue by adding this here since most of us don't really have UM active while developing.

Reply:
Good point. Removed.